gapic/search_client.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
```

Comment:
Unwanted change. Can we undo this?

You can do `git checkout` on specific files to avoid uploading such unwanted changes